pkg/envoyfilters/testdata/ingressEnvoyFilterSpecWithOneAllowRule.yaml

```diff
@@ -0,0 +1,44 @@
+configPatches:
+- applyTo: NETWORK_FILTER
+  match:
+    context: GATEWAY
+    listener:
+      filterChain:
+        sni: '*.ingress.testseed.dev.ske.eu01.stackit.cloud'
+  patch:
+    operation: INSERT_FIRST
+    value:
+      name: acl-ingress
+      typed_config:
+        '@type': type.googleapis.com/envoy.extensions.filters.network.rbac.v3.RBAC
+        rules:
+          action: ALLOW
+          policies:
+            project-shoot:
+              permissions:
+              - requested_server_name:
+                  suffix: -project-shoot.ingress.testseed.dev.ske.eu01.stackit.cloud
+              principals:
+              - remote_ip:
+                  address_prefix: 10.180.0.0
+                  prefix_len: 16
+              - remote_ip:
+                  address_prefix: 10.250.0.0
+                  prefix_len: 16
+              - remote_ip:
+                  address_prefix: 10.96.0.0
+                  prefix_len: 11
```

Comment:
Technically speaking, we probably don't need to allow all seed networks to talk to the ingresses for the shoot. We probably need the nodes' egress CIDRs for the shoot node logging feature, but I don't think we need the seed networks.
Should we adapt the code accordingly?

Reply:
Currently, seed networks seem not to be needed to be allowed to access ingresses.
But I don't see why we should forbid it. The purpose of the extension is not to control cluster internal traffic.
There are `networkPolicies` to do that. Or am I missing something.

Reply:
You're right, that's the purpose of this extension. So let's keep it like you implemented it.